AppMetrics/FeedbackDialog.qml

```diff
@@ -221,9 +221,9 @@ Dialog {
 
                             background: Rectangle {
                                 anchors.fill: parent
-                                color: Singletons.Config.buttonStates(control)
+                                color: Singletons.Config.buttonStates(parent)
                                 radius: app.info.properties.mainButtonRadius
-                                border.width: control.enabled ? app.info.properties.mainButtonBorderWidth : 0
+                                border.width: parent.enabled ? app.info.properties.mainButtonBorderWidth : 0
```

Comment:
Please refer to the parent element with its `id` instead of `parent`. This increases readability.